packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -262,6 +295,41 @@ class UserSessionManager implements IUserSessionManager {
     return this.storage?.get(userSessionStorageKeys.initialReferringDomain) || null;
   }
 
+  /**
+   * Fetches session tracking information from storage
+   * @returns
+   */
+  getSession(): Nullable<SessionInfo> {
+    return this.storage?.get(userSessionStorageKeys.sessionInfo) || null;
+  }
+
+  /**
+   * A function to return current session info
+   */
+  getSessionInfo(): Nullable<SessionInfo> {
+    const session: SessionTrackingInfo = {};
+    if (state.session.sessionInfo.value.autoTrack || state.session.sessionInfo.value.manualTrack) {
```

Comment:
is there a change that both autoTrack and manualTrack are set to false?

Reply:
Yes. Scenarios where both will be false:
1. auto Track is set to false through the load option and the customer hasn't started manual session.
2. After endSession() is called and page is not reloaded yet